lib/linguist/languages.yml

```diff
@@ -2668,6 +2669,16 @@ OpenSCAD:
   tm_scope: none
   ace_mode: scad
 
+OpenType Font Feature:
+  type: programming
```

Comment:
If this PR is accepted, this should be `data`, not `programming`. Same with Spline Font Data.